src/components/VirtualizedGrid/VirtualizedGrid.tsx

```diff
@@ -285,7 +292,7 @@ const VirtualizedGrid: VibeComponent<VirtualizedGridProps> = forwardRef(
                 rowCount={calcRowCount}
                 onScroll={onScrollCB}
                 onItemsRendered={onItemsRenderedCB}
-                className="virtualized-grid-scrollable-container"
+                className={cx("virtualized-grid-scrollable-container")}
```

Comment:
is the cx really needed?

Reply:
script is adding it everywhere by default - that's something which current implementation is dependent on + I believe it would be convenient if we'll just have it everywhere